src/game.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import {makeShuffledDeck} from "./utils"
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cardDeck: makeShuffledDeck(),
+      currentCards: [],
+      numOfGames: this.props.games,
+      gameStarted: false,
+      currentRound: 0,
+      currentGame: 1,
```

Comment:
This variable naming is a bit misleading. I would expect a whole game object here, instead of a game round count. I think we could rename this.